packages/ember-routing/lib/system/router.js

```diff
@@ -32,6 +32,18 @@ function K() { return this; }
 
 var slice = [].slice;
 
+
+var handledErrors = {};
```

Comment:
Actually, I'm not so sure that I can. The methods need to be accessible in `triggerEvent` and as far as I can tell, `triggerEvent` has no reference to the router.